plugins/flashrom/fu-flashrom-device.h

```diff
@@ -13,4 +13,10 @@
 G_DECLARE_FINAL_TYPE(FuFlashromDevice, fu_flashrom_device, FU, FLASHROM_DEVICE, FuUdevDevice)
 
 FuDevice *
-fu_flashrom_device_new(FuContext *ctx);
+fu_flashrom_device_new(FuContext *ctx, FuIfdRegion region);
+
+FuIfdRegion
+fu_flashrom_device_get_region(FuFlashromDevice *self);
+
+gboolean
+fu_flashrom_device_has_fn_m_me_unlock(FuFlashromDevice *self);
```

Comment:
I don't like this much, can you just do `fu_device_has_private_flag(device, FU_FLASHROM_DEVICE_FLAG_FN_M_ME_UNLOCK);` in the caller please. You can move the `FU_FLASHROM_DEVICE_FLAG_FN_M_ME_UNLOCK` to header if that helps.